add-ons/updating/tb78/convert-legacy.md

```diff
@@ -4,76 +4,63 @@ This guide is intended to help developers to port their Legacy WebExtensions to
+
 ```json
 "background": {
   "scripts": ["background-script.js"]
 }
 ```
+
 Adding this secton to `manifest.json` will cause the file `background-script.js` to be loaded and evaluated by Thunderbird. For bootstrap typed Legacy WebExtensions, the existing `bootstrap.js` script itself is a good starting point for a background script – for overlay typed Legacy WebExtensions it may be reasonable to start with an empty script and convert overlays using the guidelines below, gradually building up the background script.
 
-Contrary to the bootstrap script in legacy add-ons, the background scripts will *not* get evaluated in a browser context. Instead it is added to an HTML document (a.k.a the "background page") living in a content process, which only has access to [MailExtension APIs](https://thunderbird-webextensions.readthedocs.io/en/latest/index.html) and [some WebExtension APIs](new linkable header in API doc being added by geoff soon) inherited from the underlying Firefox code base. Any interaction with Thunderbird must occur through these APIs. Whenever code needs to be added to the background script, you need to make sure to migrate calls to XPCOM or other native Thunderbird features to these APIs.
+Contrary to the bootstrap script in legacy add-ons, the background scripts will *not* get evaluated in a privileged browser context. Instead it is added to an HTML document (a.k.a the "background page") living in a content process, which only has access to [MailExtension APIs](https://thunderbird-webextensions.readthedocs.io/en/latest/index.html) and some WebExtension APIs inherited from the underlying Firefox code base (they are listed further down on [this page](https://thunderbird-webextensions.readthedocs.io/en/latest/)). Any interaction with Thunderbird must occur through these APIs. Whenever code needs to be added to the background script, you need to make sure to migrate calls to XPCOM or other native Thunderbird features to these APIs.
 
-{% hint style="info" %} Since the WebExtension technology originates in Browser like Google Chrome and Firefox, the namespace for this new kind of API is `chrome.*` or `browser.*`which work in Thunderbird as well. For Thunderbird however the additional namespace `messenger.*` has been added. We think that is a better fit and should be used with MailExtensions, as they do not run in Browsers anyhow, if they use any of the MailExtension APIs, which only exist in Thunderbird. {% endhint %}
+{% hint style="info" %} Since the WebExtension technology originates in Browsers like Google Chrome and Firefox, the namespace for this new kind of API is `chrome.*` or `browser.*` which work in Thunderbird as well. For Thunderbird however the additional namespace `messenger.*` has been added. We think that is a better fit and should be used with MailExtensions, as they do not run in Browsers anyhow, if they use any of the MailExtension APIs, which only exist in Thunderbird. {% endhint %}
```

Comment:
When reading the hint box again, the last sentence seems too long / complex.
```suggestion
{% hint style="info" %} Since the WebExtension technology originates in Browsers like Google Chrome and Firefox, the namespace for this new kind of API is `chrome.*` or `browser.*` which work in Thunderbird as well. Thunderbird also supports `messenger.*`, which is a better fit for MailExtensions. As most MailExtensions are not supported by Browsers anyhow, it is safe to always use `messenger.*` when accessing Mail- or WebExtension APIs. {% endhint %}
```